src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -115,7 +115,11 @@
     // the `x-goog-api-client` header passed on each request. Intended for
     // use by Google-written clients.
     func (c *{@view.clientTypeName}) SetGoogleClientInfo(clientName, clientVersion string) {
-        c.xGoogHeader = fmt.Sprintf("gl-go/%s %s/%s gax/%s grpc/", version.Go(), clientName, clientVersion, gax.Version)
+        if clientName == "" {
```

Comment:
While you're here, how about changing this to SetGoogleClientVersion(version string), and putting in the gccl yourself? Or if we want more flexibility, at least rename clientName to clientToken.